### DISCUSSION TITLE
Connection blocking on specific wifi networks

### DISCUSSION THREAD

**Discussion by BLahz** [mullvad/mullvadvpn-app#8943] [Feature requests & Ideas]
## Issue report

Operating system: MacOS Big sur 11.6

App version: 2021.4

### Issue description

Hi. I'm using an unsecured wifi network at my workplace as it's the only choice here, and I always use VPN when connected to it (and in general anyway). However, there are certain cases at home when I can't use VPN, meaning I disconnect from it. The problem is, I sometimes forget to connect to the VPN again, and this morning I used the unencrypted wifi for almost an hour before I realized. Would it be possible to implement a "kill switch" that doesn't let you connect to internet on specific wifi networks unless you're connected to mullvad?

**Comment by faern** [comment]
This is almost the same thing as #1884. But kind of the other way around. Might close this as duplicate later, but keeping for now.

**Comment by Rawa** [comment]
Moving to discussions
